app/Parsers/RequestHeaderParser/RequestHeaderParser.php

```diff
@@ -13,7 +13,7 @@ class RequestHeaderParser
      *
      * @var array
      */
-    private static $allowedMethods = [
+    public static $allowedMethods = [
```

Comment:
I don't like this. Why not a simple getter?

(I know, we already have one to return imploded strings...)